queries.sql

```diff
@@ -9,26 +9,39 @@ SELECT * FROM animals WHERE (weight_kg >= 10.4 AND weight_kg <= 17.3);
 
 BEGIN;
 UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
 ROLLBACK;
 SELECT * FROM animals;
+
 BEGIN;
 UPDATE animals SET species = 'digimon' WHERE Name like '%mon';
 UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
-COMMIT;
+ROLLBACK;
 SELECT * FROM animals;
 BEGIN;
 DELETE FROM animals;
 ROLLBACK;
 SELECT * FROM animals;
+
 BEGIN;
 DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT date_deletion;
 UPDATE animals SET weight_kg = weight_kg * (-1);
-SAVEPOINT date_deletion
+ROLLBACK date_deletion;
 UPDATE animals SET weight_kg = weight_kg * (-1) WHERE (weight_kg < 0);
 COMMIT;
+
 SELECT COUNT(*) FROM animals;
 SELECT COUNT(*) FROM animals WHERE (escape_attempts = 0);
 SELECT AVG(weight_kg)  FROM animals;
 SELECT neutered,AVG(escape_attempts) AS escape FROM animals GROUP BY (neutered) ORDER BY escape DESC LIMIT 1;
 SELECT MIN(weight_kg),MAX(weight_kg), species FROM animals GROUP BY species;
-SELECT species,AVG(escape_attempts) AS escape FROM animals WHERE date_of_birth BETWEEN  '1990-01-01' AND '2000-01-01' GROUP BY (species) ;
\ No newline at end of file
+SELECT species,AVG(escape_attempts) AS escape FROM animals WHERE date_of_birth BETWEEN  '1990-01-01' AND '2000-01-01' GROUP BY (species);
+
+SELECT animals.name, owners.full_name FROM animals JOIN owners ON animals.owners_id=owners.id WHERE full_name = 'Melody Pond';
```

Comment:
This query should show the following animal names: `Charmander`, `Squirtle`, and `Blossom`. But according to your screenshot, the result is `Angemon` and `Boarmon` which belong to `Dean Winchester`. It's possible there was a mix up when setting the owner's id when updating the `animals` table. Kindly fix this issue.